src/components/TextSection.jsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import Wrapper from "./Wrapper";
+
+function TextSection() {
```

Comment:
this function should receive a color as an argument, or somehow learn about the color we are referring to on lines 13 + 14